cluster-support-bot.py

```diff
@@ -1,19 +1,35 @@
 import argparse
 import logging
 import os
+import re
 import time
 
-import hydra
+import prometheus_client
 import slack
+
+import hydra
 import telemetry
 
 
+mention_counter = prometheus_client.Counter('proactive_support_mentions',
```

Comment:
nit: `proactive_support_mentions` -> `cluster_support_mentions` and similar

Reply:
Done!